simple_robots/views.py

```diff
@@ -10,9 +10,10 @@ class ServeRobotsView(TemplateView):
     content_type = "text/plain"
 
     def get_template_names(self):
+        robots_allow_host_setting = getattr(settings, ROBOTS_ALLOW_HOST_SETTING, None)
         if (
-            getattr(settings, ROBOTS_ALLOW_HOST_SETTING, None)
-            == self.request.get_host()
+            robots_allow_host_setting is not None
+            and self.request.get_host() in robots_allow_host_setting
```

Comment:
Suggestion: I'd recommend using Django's `is_same_domain` util, as it nicely allows someone also specifying wildcards for domains should they wish. It's the same method used for `ALLOWED_HOSTS` and the [CSRF middleware](https://github.com/django/django/blob/main/django/middleware/csrf.py#L289).